swagger.yml

```diff
@@ -1739,6 +1761,21 @@ components:
         name:
           type: string
 
+    KeycloakWebhook:
+      type: object
+      description: Webhook from etra cloud listener keycloak plugin
+      properties:
+        resourceType:
+          type: string
+        operationType:
+          type: string
+        userId:
+          type: string
+        groupId:
+          type: string
+        creatorId:
+          type: string
```

Comment:
```suggestion
          type: string
          format: uuid
```

Reply:
User and group ids are constructed (in plugin) by parsing keycloak resource paths. There is no 100% guarantee that this method will yield uuids in all cases. Thats why it's better to use string rather than crash.